proto/cosmos/base/snapshots/v1beta1/snapshot.proto

```diff
@@ -17,4 +17,41 @@ message Snapshot {
 // Metadata contains SDK-specific snapshot metadata.
 message Metadata {
   repeated bytes chunk_hashes = 1; // SHA-256 chunk hashes
-}
\ No newline at end of file
+}
+
+// SnapshotItem is an item contained in a rootmulti.Store snapshot.
+message SnapshotItem {
```

Comment:
note: this is technically proto breaking. Want to do a sanity check on if its okay.

Reply:
yeah, you mean changing of namespace right?